### DISCUSSION TITLE
Add the current date in the prompt

### DISCUSSION THREAD

**Discussion by heyalexnet** [brianpetro/obsidian-smart-connections#405] [Ideas]
Hello! It would be a nice addition to systematically add the current date and time at the beginning of each prompt.

Currently I have to do it manually. For example: "based on my notes, what the status of my work for today?" -> I have to prefix with "We are 2024-01-08, based...". 

(I understand, you might disagree with "systematically")

**Comment by brianpetro** [comment]
@ineation interesting idea.

In `v2.1`, there will be the option to specify a note to use as a system prompt. 

So having some sort of symbol like `{{INSERT DATE}}` that's replaced at runtime might work for this.

Does that make sense? What do you think?

Thanks for your help!
🌴

**Reply by ericmburgess** [reply]
This would be fantastic. If the symbol inside `{{ }}` is treated as a template to be expanded (looked up in the `Templates` folder), that would be extremely flexible and useful.

**Reply by brianpetro** [reply]
Hey @ericmburgess 

Thanks for the feedback. You might be interested in https://github.com/brianpetro/obsidian-smart-templates

It uses the `{{ }}` syntax for making Smart Templates. The plan is to integrate it further into Smart Connections and the Smart Chat.

While it's not ready to be used as described above, you might want to keep an eye on the progress 🌴
